app/models/manageiq/providers/vmware/infra_manager/ovf_service.rb

```diff
@@ -0,0 +1,150 @@
+class ManageIQ::Providers::Vmware::InfraManager::OvfService < ServiceGeneric
+  delegate :template, :manager, :to => :service_template, :allow_nil => true
+
+  # A chance for taking options from automate script to override options from a service dialog
+  def preprocess(action, new_options = {})
+    return unless action == ResourceAction::PROVISION
+
+    if new_options.present?
+      _log.info("Override with new options:")
+      $log.log_hashes(new_options)
+    end
+
+    save_action_options(action, new_options)
+  end
+
+  def execute(action)
+    return unless action == ResourceAction::PROVISION
+
+    library_item_deploy_queue
+  end
+
+  def library_item_deploy_queue
+    task_options = {:action => "Deploying VMware Content Library Item", :userid => "system"}
+    queue_options = {
+      :class_name  => self.class.name,
+      :instance_id => id,
+      :method_name => "library_item_deploy",
+      :args        => {},
+      :role        => "ems_operations",
+      #:queue_name  => manager.queue_name_for_ems_operations,
+      :zone        => manager.my_zone
+    }
+
+    task_id = MiqTask.generic_action_with_callback(task_options, queue_options)
+    task = MiqTask.wait_for_taskid(task_id)
+    raise task.message unless task.status_ok?
+  end
+
+  def library_item_deploy(_options)
+    _log.info("OVF template provisioning with template ID: [#{template.id}] name:[#{template.name}] was initiated.")
+    opts = provision_options
+    _log.info("VMware Content Library OVF Tempalte provisioning with options:")
+    $log.log_hashes(opts)
```

Comment:
Should this be `_log` instead of `$log`?

Reply:
This has been discussed [here](https://github.com/ManageIQ/manageiq/pull/14878#discussion_r113572760) a while ago.

Reply:
Oh okay interesting, this is fine for now but I think especially with us having container logs (and even journald logs) aggregated and searched over it is helpful to have multi-line output like this actually associated with the first log message.

We hit this with our typical:
```
_log.error("Failed to whatever: #{err}")
_log.log_backtrace(err)
```

It made it harder to search for the error in kibana, then go and try to find the backtrace.

It was better to (in this case) to something like ` _log.info("VMware Content Library OVF Tempalte provisioning with options:\n#{log_hashes(opts)}")`

Reply:
```
[----] I, [2020-09-23T14:02:47.185056 #97828:3fd032c3205c]  INFO -- : Q-task_id([r200_service_template_provision_task_154])
---
template_id: '33'
dialog_id: '5'
fqname: "/Service/Generic/StateMachines/GenericLifecycle/provision"
vm_name: w_vm_2
accept_all_EULA: true
resource_pool_id: 2
ems_folder_id: 30
[----] I, [2020-09-23T14:02:47.185127 #97828:3fd032c3205c]  INFO -- : Q-task_id([r200_service_template_provision_task_154]) MIQ(ManageIQ::Providers::Vmware::InfraManager::OvfService#deploy_library_item) VMware Content Library OVF Tempalte provisioning with options:
true
```
Check out the result. Haha.

Reply:
Haha its `true`, can't argue with that.  This was assuming we had a method `log_hashes` which returns the `\n` delineated string